FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -175,6 +182,10 @@ paths:
       security:
       - ComponentRead:
         - fraihmwork.component.read
+      - ComponentWrite:
```

Comment:
Should write and admin be added to GET endpoints?

Reply:
This is so write and admin scopes can read these implicitly